app/models/Organisation.js

```diff
@@ -87,29 +100,24 @@ const orgSchema = new Schema({
     },
     chattingPlatform: [
       {
+        _id: false,
         link: {
           type: String
         }
       }
     ]
   },
   adminInfo: {
-    type: Object,
-    required: true,
-    validate (adminInfo) {
-      if (validator.isEmpty(adminInfo)) {
-        throw new Error('Admin info is required!')
-      }
-    }
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
   },
   moderatorInfo: {
-    type: Object,
-    required: true,
-    validate (adminInfo) {
-      if (validator.isEmpty(adminInfo)) {
-        throw new Error('Admin info is required!')
-      }
-    }
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  isArchived: {
```

Comment:
why do we need this `isArchived`

Reply:
@devesh-verma I have added that if an admin wants then they can either delete the organization completely or keep that in archived mode.
What say @devesh-verma ?

Reply:
cool